rent_gpu/requestor/api/templates/offer-details.html

```diff
@@ -42,13 +42,8 @@ <h2>Provider status</h2>
             .accordion {
               background-color: #eee;
               color: #444;
-              cursor: pointer;
-              padding: 10px;
-              width: 100%;
-              border: none;
-              text-align: left;
+              {#text-align: left;#}
```

Comment:
To remove.

Reply:
Sorry - I forgot to do it :pray:. Now it is done.